recipes-wigwag/relay-term/relay-term_0.0.1.bb

```diff
@@ -26,16 +32,19 @@ ${PN}-watcher.path"
 
 SYSTEMD_AUTO_ENABLE_${PN} = "enable"
 
-
-RDEPENDS_${PN} += "bash"
-
+DEPENDS ="nodejs"
+RDEPENDS_${PN} += "bash nodejs"
 
 do_install() {
+    cd ${S}
     install -d ${D}${systemd_system_unitdir}
     install -m 755 ${WORKDIR}/${RT_SERVICE_FILE} ${D}${systemd_system_unitdir}/${RT_SERVICE_FILE}
     install -m 755 ${WORKDIR}/${PN}-watcher.service ${D}${systemd_system_unitdir}/${PN}-watcher.service
     install -m 755 ${WORKDIR}/${PN}-watcher.path ${D}${systemd_system_unitdir}/${PN}-watcher.path
-}
-
 
+    install -d ${D}/wigwag/devicejs-core-modules/node_modules
+    cp -r ${S}/node_modules ${D}/wigwag/devicejs-core-modules/node_modules
 
+    install -d ${D}/wigwag/wigwag-core-modules
+    cp -r ${S} ${D}/wigwag/wigwag-core-modules/relay-term
```

Comment:
What is ${S} pointing to?  Does this copy all of edge-node-modules into the relay-term folder, for example DevStateManater, LEDController, RelayStatsSender, etc.?